src/__tests__/ParseObject-test.js

```diff
@@ -2418,43 +2417,43 @@ describe('ParseObject', () => {
       done();
     });
     jest.runAllTicks();
-    await flushPromises();
-
-    xhrs[0].responseText = JSON.stringify([
-      { success: { objectId: 'pid0' } },
-      { success: { objectId: 'pid1' } },
-      { success: { objectId: 'pid2' } },
-      { success: { objectId: 'pid3' } },
-      { success: { objectId: 'pid4' } },
-      { success: { objectId: 'pid5' } },
-      { success: { objectId: 'pid6' } },
-      { success: { objectId: 'pid7' } },
-      { success: { objectId: 'pid8' } },
-      { success: { objectId: 'pid9' } },
-      { success: { objectId: 'pid10' } },
-      { success: { objectId: 'pid11' } },
-      { success: { objectId: 'pid12' } },
-      { success: { objectId: 'pid13' } },
-      { success: { objectId: 'pid14' } },
-      { success: { objectId: 'pid15' } },
-      { success: { objectId: 'pid16' } },
-      { success: { objectId: 'pid17' } },
-      { success: { objectId: 'pid18' } },
-      { success: { objectId: 'pid19' } },
-    ]);
-    xhrs[0].onreadystatechange();
-    jest.runAllTicks();
-    await flushPromises();
+    flushPromises().then(async () => {
```

Comment:
Why is this a chained promise and not also an await?

Reply:
I didn’t want to go though callback hell.

Reply:
If it's a lot of work I guess we can leave it as is.

Reply:
This is good to merge.